lamindb/core/loaders.py

```diff
@@ -107,9 +108,17 @@ def load_json(path: UPathStr) -> dict:
         data = json.load(f)
     return data
 
+def load_yaml(path: UPathStr) -> dict:
+    """Load `.yaml` to `dict`."""
+    import yaml
```

Comment:
Maybe except `ImportError` and log a warning that the package is not installed and return a path.